src/main/java/cz/muni/fi/pa165/bluebat/entity/Trip.java

```diff
@@ -28,31 +19,30 @@
 @Entity
 @Setter
 @ToString
+@Getter
 public class Trip {
 
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
-    @Getter
     private Long id;
 
-    @Getter
+
     private String name;
```

Comment:
Name bych označil jako NotBlanka a NotNull, přece jen to je důležité, aby to bylo jako nějaký identifikátor daného Tripu. Možná bych zvážil použití @Column(unique = true) a podle toho upravení hash a equal metod

Reply:
Ako nevidim problem z primarnym klucom z tromi premenimy ja by som to name vyhodil skor ak dal unique, ale tak mi bol dany trip a mam meno.  Ja som to skor bral ako description. @NotBlank pridam dik.